src/routes/style.css

```diff
@@ -29,8 +30,19 @@ a:hover {
 }
 
 h1 {
-	font-size: 3.2em;
-	line-height: 1.1;
+	/* Logo font */
+	font-family: 'Ubuntu', sans-serif;
+	font-weight: 700;
+	font-size: 70px;
+	line-height: 80px;
+	display: flex;
+	align-items: center;
+
+	/* White */
+	color: #EEEEEE;
+
+	/* Logo Shadow */
+	text-shadow: 0px 1.37056px 0px #B1B1B1;
```

Comment:
This gets rendered with the wrong font weight in firefox on windows and I have no clue why. I atomized the issue in `static/Logo-Test.html`, which gets rendered the right way.
There seems to be some interference or some svete-scope stuff I don't understand.
Input needed.
![image](https://github.com/TuDU-SE/TuDu-Frontend/assets/65495509/09cd0a33-d0f7-4d72-a24f-cba2796b92c7)
![image](https://github.com/TuDU-SE/TuDu-Frontend/assets/65495509/5221d90a-1a66-4ea0-aee2-9c51ebea247d)

Reply:
I have the same, i don't know why. I would approve it nevertheless.

Reply:
I am not satisfied with this thinner font. Have you tested other browsers?

Reply:
I just tried Chromium/edge, same results

Reply:
For me it works in chrome (left Firefox/ right Chrome): 
![grafik](https://github.com/TuDU-SE/TuDu-Frontend/assets/46647854/3db97868-f69e-4511-80e5-4ab6b775162f)

Reply:
Please compare with `/Logo-Test.html`.

Reply:
I seem to have solved the issue in https://github.com/TuDU-SE/TuDu-Frontend/pull/9/commits/eae12559669f5f0c3a1d29a38ea38c92ef605bfc.